### PR TITLE
diagnostics: update check definitions to 4.8.0

### PR DESCRIPTION
**Pull request by xginn8**
Depends-on: https://github.com/balena-io/resin-proxy/pull/410
Change-type: patch
Signed-off-by: Matthew McGinn <matthew@balena.io>

### CODE REVIEW

**Comment by imrehg** [comment]
I think there is a heads up, not to merge anything without pinging Mike, can link to it later, but mentioning in case. I *think* this is still in effect.

**Comment by lurch** [comment]
There's a "closing bracket" at `or logic built in)` which doesn't seem to have a matching "opening bracket" ?  :eyes:

File: pages/reference/diagnostics.md [hunk 1]
```diff
@@ -7,13 +7,16 @@ A `check` in this context is defined as a function that returns a result (good/b
 command is simply a data collection tool without any filtering or logic built in). Checks are intended to be used by
```
Comment (imrehg) [inline pages/reference/diagnostics.md]: Is this a good place to put the bracket or better to dispose of the bracket in this section entirely?
```suggestion
command is simply a data collection tool (without any filtering or logic built in). Checks are intended to be used by
```
based on @lurch's comment?

Comment (xginn8) [reply pages/reference/diagnostics.md]: Nice catch @lurch, I've opened a PR in the repo where we source this doc, and will pull that in once it's been approved.

**Pull request merged**